modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp

```diff
@@ -141,6 +149,10 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 
 <liferay-portlet:resourceURL copyCurrentRenderParameters="<%= false %>" id="saveFormInstance" var="autoSaveFormInstanceURL" />
 
+<div class="hide" id="viewFormInstanceRecords">
+	<liferay-util:include page="/admin/list_form_instance_records.jsp" servletContext="<%= application %>" />
```

Comment:
never seen `servletContext` I need to understand what this is for.

Reply:
"The servletContext refers to the request context that the included JSP should use. Passing <%= application %> to this attribute lets the included JSP use the same request object as other objects that might be set in the prior JSP. "

https://help.liferay.com/hc/pt/articles/360023185391-Using-Liferay-Util-Include-

Example of usage [here](https://github.com/liferay/liferay-portal/blob/43f034138129c4601c1dd0cdf076d8cfc4e99c01/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view.jsp#L29).